### PR TITLE
new lab member - MTi

### PR DESCRIPTION
**Pull request by mtitov**

### CODE REVIEW

File: _people/mikhail_titov.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: "Research Associate"
+permalink: /people/mikhail_titov
+contacts:
+  email: mikhail.titov@rutgers.edu
+  office: CoRE 707
+  github: https://github.com/mtitov
+  linkedin: https://linkedin.com/in/mikhail-a-titov
+---
+
+Mikhail is a Research Associate at the Rutgers University. He pursued his Ph.D.
```
Comment (iparask) [inline _people/mikhail_titov.md]:
```suggestion
Mikhail is a Research Associate at Rutgers University. He pursued his Ph.D. 
```

File: _people/mikhail_titov.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Mikhail is a Research Associate at the Rutgers University. He pursued his Ph.D. 
+degree in Computer Science at the University of Texas at Arlington in 2016, 
+under the supervision of Dr. Gergely Záruba and Dr. Kaushik De. His doctoral 
+research was focused on applying data mining and machine learning techniques 
+and methods to the workload management system PanDA (the Production and 
+Distributed Analysis system) to investigate correlations and similarities 
+between data objects (involved entities, e.g., inter-user, user-data 
+relationships) for efficient data distribution and data caching strategies. 
+In 2007, he received his Combined B.S./M.S. degree in Computer Engineering
```
Comment (iparask) [inline _people/mikhail_titov.md]:
```suggestion
In 2007, he received his combined B.S./M.S. degree in Computer Engineering 
```

File: _people/mikhail_titov.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+research was focused on applying data mining and machine learning techniques 
+and methods to the workload management system PanDA (the Production and 
+Distributed Analysis system) to investigate correlations and similarities 
+between data objects (involved entities, e.g., inter-user, user-data 
+relationships) for efficient data distribution and data caching strategies. 
+In 2007, he received his Combined B.S./M.S. degree in Computer Engineering 
+(Computing and Information Science) at the Moscow Engineering and Physics 
+Institute (NRNU "MEPhI", Moscow, Russia). Since 2008, he is an Associated member 
+of the personnel (Scientist) at the European Organization for Nuclear Research 
+(CERN, Geneva, Switzerland).
```
Comment (iparask) [inline _people/mikhail_titov.md:28]: Do you mean "... an associate staff member (scientist)at CERN, the European Organization for Nuclear Research, at Geneva, Switzerland."?

File: _people/mikhail_titov.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+Research interest is data analysis with a focus on analysis of metadata gathered
```
Comment (iparask) [inline _people/mikhail_titov.md]:
```suggestion
His research interest is data analysis with a focus on analysis of metadata gathered 
```

File: _people/mikhail_titov.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+In 2007, he received his Combined B.S./M.S. degree in Computer Engineering 
+(Computing and Information Science) at the Moscow Engineering and Physics 
+Institute (NRNU "MEPhI", Moscow, Russia). Since 2008, he is an Associated member 
+of the personnel (Scientist) at the European Organization for Nuclear Research 
+(CERN, Geneva, Switzerland).
+
+Research interest is data analysis with a focus on analysis of metadata gathered 
+from distributed processing systems. This process of data analytics 
+(descriptive and predictive) is to reveal and explore implicit relationships 
+and inter-dependencies between data records and analysis entities. That is the
```
Comment (iparask) [inline _people/mikhail_titov.md]: May I suggest:
" Data analytics (descriptive and predictive) reveal and explore the implicit relationships and inter-dependencies..."

I am not sure what "analysis entities" are. I do not think there is a need to change it if there is no other way to say it.

File: _people/mikhail_titov.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+of the personnel (Scientist) at the European Organization for Nuclear Research 
+(CERN, Geneva, Switzerland).
+
+Research interest is data analysis with a focus on analysis of metadata gathered 
+from distributed processing systems. This process of data analytics 
+(descriptive and predictive) is to reveal and explore implicit relationships 
+and inter-dependencies between data records and analysis entities. That is the 
+core which interests are built around, and which include research areas such as 
+grid computing, cloud computing, high performance computing, data mining, 
+machine learning, modeling and simulation.
```
Comment (iparask) [inline _people/mikhail_titov.md:36]: How about: "In addition, his research interests include research areas such as grid computing, cloud computing, high performance computing, data mining, machine learning, modeling and simulation."?

File: _people/mikhail_titov.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+Mikhail has joined the RADICAL group in February 2020.
```
Comment (iparask) [inline _people/mikhail_titov.md]:
```suggestion
Mikhail joined the RADICAL group in February 2020.
```

**Review by iparask — COMMENTED** [review]
@mtitov, I left a few comments. Let me know what you think so that we can move forward with your page

File: _people/mikhail_titov.md (hunk 3 above)
Comment (mtitov) [reply _people/mikhail_titov.md:28]: that is actually that way how I'm affiliated to CERN (it is "Associated member of the personnel"), staff is who actually employed by CERN

File: _people/mikhail_titov.md (hunk 5 above)
Comment (mtitov) [reply _people/mikhail_titov.md]: I would like to keep it as it is. It is connected to the previous sentence and to the following one (which is in your comment below), that's why it is written that way

**Comment by mtitov** [comment]
Thank you @iparask ! updated it accordingly

**Review by iparask — APPROVED** [review]

**Pull request merged**